app/Http/Controllers/CompetitionsController.php

```diff
@@ -59,7 +60,13 @@ public function store(CompetitionRequest $request)
      */
     public function show(Competition $competition)
     {
-        return view('competitions.show', compact('competition'));
+        $bracket = Competition::getBracket($competition->id);
```

Comment:
@weerd it's "implemented" in the since that all it does is get all users from the `competition_users` table that have that competition id...but that needs to be updated once we have brackets. and :+1: to the pluralization.